check_runner/__init__.py

```diff
@@ -0,0 +1,153 @@
+"""
```

Comment:
`check_runner` is a script amongst the other scripts (although it's not a check plugin) of `monitoring-scripts`. It doesn't warrant having its own separated package. I'd recommend looking into fitting it in `monitoring-scripts/check_runner.py`